docs/integrations/tools/index.md

```diff
@@ -13,7 +13,7 @@ The table below provides an index of custom tool packages. The columns contain:
 
 | Package Name | Description | Owner | Support Contact |  
 |-|-|-|-|
-|||||
+| promptflow-azure-ai-language | Collection of Azure AI Language Prompt flow tools. | Sean Murray | murraysean@microsoft.com |
```

Comment:
@murraysean for 'support contact' column, please kindly use a group email for better support and issue routing.